Mention generate constructor in null safety FAQ (#4368)
dart-lang/site-www: src/null-safety/faq.md

];
{% endprettify %}

> If you need to generate a fixed-length list,
> you can instead use the [`List.generate`][] constructor
> and set the `growable` parameter to `false`:

Reviewer: **suggestion:** Tighten this language. The conditional clause is not needed. You can just state this in the "why, how" construction. If they don't need it, they can move past it.
```suggestion
To generate a fixed-length list,
use the [`List.generate`][] constructor
with the `growable` parameter set to `false`:
```
